packages/cf.js/src/types.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./simple-types";
```

Comment:
I don't think we should have a `types.ts` file. Is this meant to mimic `ethers.types`?

Reply:
Why not? Where do we put `Address`, `H256`, `ABIEncoding` typedefs?